internal/sidekick/internal/rust/templates/crate/src/transport.rs.mustache

```diff
@@ -94,6 +94,7 @@ impl super::stub::{{Codec.Name}} for {{Codec.Name}} {
             {{^Codec.HasVariablePath}}
             let path = "{{Codec.PathFmt}}".to_string();
             {{/Codec.HasVariablePath}}
+            let path_template = "{{Codec.PathTemplate}}".to_string();
```

Comment:
```suggestion
            let path_template = "{{Codec.PathTemplate}}";
```

And defer the allocation (if needed) until the `set_path_template()` call.  Is that possible?

Reply:
for the "if needed" bit... passing around a `&'static str` is pretty much free. It is like passing around a pointer. We can do that even when the contents of the `&'static str` are not needed.

We want to avoid creating a `String` unless that `String` is needed.